data/bss-factory-sets.json

```diff
@@ -0,0 +1,5689 @@
+    ]
+  },
+  "basculegion": {
+    "weight": 7,
+    "sets": [
+      {
+        "species": "basculegion",
+        "weight": 20,
+        "moves": [
+          ["lastrespects"],
+          ["wavecrash"],
+          ["aquajet", "sleeptalk"],
+          ["flipturn"]
+        ],
+        "gender": "m",
+        "item": ["choicescarf"],
+        "nature": "adamant",
+        "evs": [0, 252, 0, 4, 0, 252],
+        "teratype": ["fairy", "normal", "fighting", "water", "ghost"],
+        "ability": "adaptability"
+      },
+      {
+        "species": "basculegion",
+        "weight": 10,
+        "moves": [["lastrespects"], ["wavecrash"], ["flipturn"], ["terablast"]],
+        "gender": "m",
+        "item": ["choicescarf"],
+        "nature": "adamant",
+        "evs": [0, 252, 0, 4, 0, 252],
+        "teratype": ["fairy", "fighting"],
+        "wantsTera": true,
+        "ability": "adaptability"
+      },
+      {
+        "species": "basculegion",
+        "weight": 40,
+        "moves": [
+          ["wavecrash"],
+          ["aquajet"],
+          ["agility", "endeavor"],
+          ["lastrespects"]
+        ],
+        "gender": "m",
+        "item": ["focussash"],
+        "nature": "adamant",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["normal", "ghost", "water"],
+        "ability": "adaptability"
+      },
+      {
+        "species": "basculegion",
+        "weight": 20,
+        "moves": [
+          ["wavecrash"],
+          ["aquajet"],
+          ["agility", "terablast"],
+          ["lastrespects"]
+        ],
+        "gender": "m",
+        "item": ["focussash"],
+        "nature": "adamant",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["fairy"],
+        "wantsTera": true,
+        "ability": "adaptability"
+      },
+      {
+        "species": "basculegion",
+        "weight": 5,
+        "moves": [["wavecrash"], ["aquajet"], ["substitute"], ["lastrespects"]],
+        "gender": "m",
+        "item": ["brightpowder"],
+        "nature": "adamant",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["normal", "ghost", "water"],
+        "ability": "adaptability"
+      },
+      {
+        "species": "basculegion",
+        "weight": 5,
+        "moves": [
+          ["wavecrash"],
+          ["aquajet"],
+          ["flipturn", "liquidation", "substitute"],
+          ["lastrespects"]
+        ],
+        "gender": "m",
+        "item": ["lifeorb"],
+        "nature": "adamant",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["water"],
+        "ability": "swiftswim"
+      }
+    ]
+  },
+  "corviknight": {
+    "weight": 6,
+    "sets": [
+      {
+        "species": "corviknight",
+        "weight": 50,
+        "moves": [
+          ["uturn"],
+          ["roost"],
+          ["bodypress", "ironhead"],
+          ["taunt", "bravebird"]
+        ],
+        "item": ["rockyhelmet", "leftovers"],
+        "nature": "impish",
+        "evs": [252, 0, 252, 0, 0, 4],
```

Comment:
Please don't put the 4 in Speed on max/max sets in randoms. Move it elsewhere.

Also applies to other corv sets.